R/url.r

```diff
@@ -107,6 +107,10 @@ build_url <- function(url) {
     port <- NULL
   }
 
+  if (length(url$path) > 1) {
+    url$path <- paste(url$path, collapse = "/")
+  }
+
   path <- url$path
```

Comment:
I think you should just skip the if - there's no harm to pasting if length is 1. Then you can combine with the next line.